content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -601,6 +603,17 @@ This code set contains the area of the body the surgery procedure is being execu
       "display": "Left",
       "userSelected": true
     }
+    
+##### Code Set 14229 Scheduling Appointment Cancellation Reason 
```

Comment:
Shouldn't this be `Appointment Cancelation Reason`?

Reply:
Actually in all of the places there is "cancellation", which is why I am a little hesitant.
Aren't both the spellings grammatically correct?  @Coffeevee

Reply:
Yes, both spellings are technically gramatically correct, but I've been using 'cancelation reason' to stay consistent with the FHIR field

Reply:
I can see in the hl7 documentation it is LL in many places ,
https://www.hl7.org/fhir/r4/valueset-appointment-cancellation-reason.html
wouldn't that be inconsistent as well?

Reply:
Gotcha. Based on this and the fact that most people use the spelling with 'LL' anyways, I think I'm fine with leaving it this way then.